docs/specifications/tests/Zone-TP/zone09.md

```diff
@@ -100,7 +100,8 @@ correct DNS response for an authoritative name server.
 
 Message Tag                | Level | Arguments               | Message ID for message tag
 :--------------------------|:------|:------------------------|:--------------------------------------------
-Z09_INCONSISTENT_MX_RRSETS |WARNING|                         | The name servers return different MX RRsets.
+Z09_INCONSISTENT_MX        |WARNING|                         | Some name servers return an MX RRset while others return no MX.
```

Comment:
suggestion: "while others return no MX" -> "while others return none"

Reply:
Yes, that works too. I'll update. Please re-review and mark comments as resolved if possible.